hikari/embeds.py

```diff
@@ -991,3 +991,30 @@ def __eq__(self, other: typing.Any) -> bool:
             else:
                 return True
         return False
+
+    def total_length(self) -> int:
+        """Return the total length/character count of this embed.
+
+        Useful for determining if the embed exceeds Discord's 6000 character
```

Comment:
Do we really want to document what the current maximum is?

Docstring could just be left to:
```
"""Get the total character count of the embed, including fields, author and footer.
""""
```